fedimint-ui/apps/gateway-ui/src/bitcoin.types.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Standard Bitcoin transaction types
+ * We could package this into a TS types library for Bitcoin
```

Comment:
https://github.com/ts-bitcoin/ts-bitcoin

😵☠️

Reply:
Why link to a BSV types lib? :/